map/routing_mark.cpp

```diff
@@ -192,7 +192,7 @@ drape_ptr<df::UserPointMark::SymbolNameZoomInfo> RouteMarkPoint::GetSymbolNames(
       case 0: name = "route-point-a"; break;
       case 1: name = "route-point-b"; break;
       case 2: name = "route-point-c"; break;
-      default: name = ""; break;
+      default: name = "route-point-c"; break;  /* This should become the "+" icon for the rest of the intermediate steps */
```

Comment:
We use // style comments in C++. They allow to quickly temporarily comment a block of code (including // comments) with /* */

Reply:
Please also add
// TODO: Properly add icons for other letters/numbers after C.